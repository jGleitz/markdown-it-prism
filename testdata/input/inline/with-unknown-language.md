`System.out.println("bar")`{language=this-is-not-a-language}
