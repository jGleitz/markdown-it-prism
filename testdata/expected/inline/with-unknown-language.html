<p><code class="language-this-is-not-a-language">System.out.println(&quot;bar&quot;)</code></p>
